web-handlers.go

```diff
@@ -36,7 +36,12 @@ import (
 // isJWTReqAuthenticated validates if any incoming request to be a
 // valid JWT authenticated request.
 func isJWTReqAuthenticated(req *http.Request) bool {
-	jwt := initJWT()
+	jwt, err := newJWT()
+	if err != nil {
+		errorIf(err, "error in getting new JWT")
```

Comment:
we don't have say error as 'error' its redundant. We can say like `Unable to initialize a new JWT`